pkg/kubectl/describe.go

```diff
@@ -878,10 +878,33 @@ func describeNode(node *api.Node, pods []*api.Pod, events *api.EventList) (strin
 			fmt.Fprintf(out, "ExternalID:\t%s\n", node.Spec.ExternalID)
 		}
 		fmt.Fprintf(out, "Pods:\t(%d in total)\n", len(pods))
-		fmt.Fprint(out, "  Namespace\tName\n")
+		fmt.Fprint(out, "  Namespace\tName\t\tCPU(milliCPU)\t\tMemory(bytes)\n")
+		totalMilliCPU := int64(0)
+		totalMemory := int64(0)
+		fractionPodCPU := float64(0)
+		fractionPodMemory := float64(0)
+		fractionTotalCPU := float64(0)
+		fractionTotalMemory := float64(0)
 		for _, pod := range pods {
-			fmt.Fprintf(out, "  %s\t%s\n", pod.Namespace, pod.Name)
+			podTotalMilliCPU := int64(0)
+			podTotalMemory := int64(0)
+
+			for ix := range pod.Spec.Containers {
```

Comment:
@derekwaynecarr Do we expect this to work for clusters with security policy enabled? An arbitrary client might not be able to read all the pods.

That issue doesn't have to block this PR, since the existing code has the same problem, but it would be nice if this at least gracefully handled that case.